resources/fluid.py

```diff
@@ -7,16 +7,57 @@
 from scp.ethylene_glycol import EthyleneGlycol
 from scp.methyl_alcohol import MethylAlcohol
 from scp.propylene_glycol import PropyleneGlycol
-from scp.water import Water
+from scp.water import Water as ScpWater
 
 
 class FluidType(Enum):
-    EthylAlcohol = auto()
-    EthyleneGlycol = auto()
-    MethylAlcohol = auto()
-    PropyleneGlycol = auto()
-    Water = auto()
+    EthylAlcohol = 1
```

Comment:
Remove the `auto` so that we can quickly convert from the schedule magic number to the fluid type. The requirement that these numbers match up with numbers elsewhere isn't great, but it does work.